lnrpc/lightning.proto

```diff
@@ -755,6 +755,19 @@ message GetTransactionsRequest {
 
     // An optional filter to only include transactions relevant to an account.
     string account = 3;
+
+    /*
+    The index of a transaction that will be used in a query to determine which
+    transaction should be returned in the response, where the index_offset is
+    excluded.
+    */
+    uint64 index_offset = 4;
```

Comment:
To be able to correlate this value in the response and to know what next index to choose, I think we should also include `last_index_offset` and `first_index_offset` in the **response**. See `ListInvoiceResponse` for example.